.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: python
 cache: pip
 python:
-  - "nightly" # currently points to 3.6-dev
+  # see https://docs.travis-ci.com/user/languages/python/ for options
+  - "nightly"
```

Comment:
I'm not sure on what travis-ci considers nightly, but since no tags nor branches exist for 3.7 yet (nor will they for a little bit), I can only assume that they mean `tip` (which is https://hg.python.org/cpython/shortlog/default ) 

Right now, tip is the same as 3.6 since 3.6 is still in beta mode and 3.7 hasn't had any PEPs planned against it yet